lib/screens/landing_page/land_page.dart

```diff
@@ -0,0 +1,216 @@
+import 'package:flutter/material.dart';
```

Comment:
Should this file stay? Since you seem to have decided to use landing_page.dart

Reply:
it will stay